CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## master / unreleased
+* [CHANGE] Code clean up and update documentation
```

Comment:
This doesn't introduce any user facing change so no need changelog.

Reply:
Done